django_refugeerights/subscription/tests.py

```diff
@@ -85,6 +85,24 @@ def test_create_subscription(self):
         self.assertEqual(d.lang, "en_GB")
         self.assertEqual(d.schedule.id, self.schedule.id)
 
+    def test_switch_subscription(self):
+        response = self.client.get('/subscription/switch_subscription/',
+                                   {"to_addr": "+278888"},
+                                   content_type='application/json')
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(len(response.data), 1)
+        sub = response.data[0]
+        self.assertEqual(sub["active"], True)
+        patch_data = {"active": False}
+        patch = self.client.patch(
```

Comment:
You can remove this whole second half of the test -- you only want to test doing a `post` for now.